clever/launch/main_camera.launch

```diff
@@ -1,20 +1,19 @@
 <launch>
-    <!-- Camera position and orientation are represented by base_link -> main_camera_optical transform -->
-    <!-- static_transform_publisher arguments: x y z yaw pitch roll frame_id child_frame_id -->
-
     <!-- article about camera setup: https://clever.coex.tech/camera_frame -->
 
-    <!-- camera is oriented downward, camera cable goes backward [option 1] -->
-    <node pkg="tf2_ros" type="static_transform_publisher" name="main_camera_frame" args="0.05 0 -0.07 -1.5707963 0 3.1415926 base_link main_camera_optical"/>
+    <arg name="orientation_z" default="down"/> <!-- direction the camera points: down, up -->
+    <arg name="orientation_y" default="backward"/> <!-- direction the camera cable points: backward, forward -->
```

Comment:
Well, I've said it before and I'll say it again: people will make typos, and that's especially true for not very simple words like "forward" and "backward". And those who will make those typos won't even consider copying and pasting values (obviously).

Another thing that a user will probably be concerned about is why `orientation_x` is missing - surely a drone may fly with its main camera pointing forward?

Reply:
`orientation_x` is not necessary, you only need two vectors two define an orientation.

Reply:
Okay, yeah, it's just that `orientation_z` and `orientation_y` names are confusing. Have you looked at other APIs that allow setting camera direction? One good example is [the glm (OpenGL mathematics) library](http://glm.g-truc.net/0.9.9/api/a00247.html#gaa64aa951a0e99136bba9008d2b59c78e) - among other things, it allows specifying the "up" direction of the camera.

Reply:
Maybe rename `orientation_y` to` orientation_xy`? I agree that for a person unfamiliar with our frame system, such a name will be completely incomprehensible and unusual.
With the name `orientation_z` I agree, even a new person will understand this quite.

Reply:
What is `orientation_xy`?